worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightPlatformAdapter.java

```diff
@@ -590,10 +590,11 @@ static void removeBeacon(BlockEntity beacon, LevelChunk levelChunk) {
     }
 
     static List<Entity> getEntities(LevelChunk chunk) {
-        if (PaperLib.isPaper()) {
-            return Arrays.asList(chunk.entities.getRawData());
-        }
-        return chunk.level.entityManager.getEntities(chunk.getPos());
+//        if (PaperLib.isPaper()) {
+//            return Arrays.asList(chunk.entities.getRawData());
+//        }
+//        return chunk.level.entityManager.getEntities(chunk.getPos());
+        return new ArrayList<>();
```

Comment:
This method needs to support:
- Spigot
- Paper before the new chunk system
- current Paper